_includes/exhibits.js

```diff
@@ -1968,7 +1974,7 @@ HashState.prototype = {
     var selected = this.g === g ? true : false;
     aEl = Object.assign(aEl, {
       className: selected ? 'nav-link active' : 'nav-link',
-      style: 'padding-right: 40px;',
+      style: 'padding-right: 50px; position: relative;',
```

Comment:
Give more space to "MORE" button, Compensates for extra space taken by scrollbar.